subprojects/docs/src/docs/userguide/artifactDependenciesTutorial.xml

```diff
@@ -71,44 +71,68 @@
 
     <section id="configurations">
         <title>Dependency configurations</title>
-        <para>In Gradle dependencies are grouped into <firstterm>configurations</firstterm>. A configuration is simply a named set of dependencies.
-            We will refer to them as <firstterm>dependency configurations</firstterm>. You can use them to declare the external dependencies of your project.
-            As we will see later, they are also used to declare the publications of your project.</para>
-        <para>
-            The Java plugin defines a number of standard configurations. These configurations represent the classpaths that the
-            Java plugin uses. Some are listed below, and you can find more details in <xref linkend="tab:configurations"/>.
+        <para>In Gradle dependencies are grouped into <firstterm>configurations</firstterm>. A configuration is a named set of dependencies.
```

Comment:
I'd shorten this to just 

"A Configuration is a named set of dependencies and artifacts. There are three main purposes for a Configuration:"